core/src/Helpers/StringWriterWithEncoding.cs

```diff
@@ -25,17 +25,20 @@
 //
 //------------------------------------------------------------------------------
 
+using System.Globalization;
 using System.IO;
 using System.Text;
 
 namespace Microsoft.Identity.Core.Helpers
 {
     internal class StringWriterWithEncoding : StringWriter
-    {
-        public StringWriterWithEncoding(Encoding encoding)
-        {
+    {
+
+        public StringWriterWithEncoding(Encoding encoding) 
+            : base(CultureInfo.InvariantCulture)

```

Comment:
Derived class calls base parameterless constructor implicitly, which violates CA1305. Setting the default culture works around it.